fxa-auth-db-server/index.js

```diff
@@ -117,6 +119,29 @@ function createServer(db) {
 
   api.get('/__heartbeat__', reply(db.ping))
 
+  api.put(
+    '/account/:id/device/:deviceId',
+    function (req, res, next) {
+      db.upsertDevice(req.params.uid, req.params.deviceId, req.body)
```

Comment:
for completeness: yes we should do this, and yes it belongs in the corresponding auth-server PR; thanks @vladikoff